infrastructure/transform/network.tf

```diff
@@ -0,0 +1,188 @@
+resource "azurerm_network_security_group" "databricks" {
+  name                = "nsg-dbks-${var.naming_suffix}"
+  resource_group_name = var.core_rg_name
+  location            = var.core_rg_location
+  tags                = var.tags
+
+  security_rule {
```

Comment:
Do we need to also allow traffic to / from the Satellite?

Reply:
Works! It's because there's a default rule that allows vnet inbound